lib/msal-node/package.json

```diff
@@ -35,8 +35,8 @@
     "lint:fix": "npm run lint -- -- --fix",
     "build:all": "npm run build:common && npm run build",
     "build:common": "cd ../msal-common && npm run build",
-    "link:localDeps": "npx lerna bootstrap --scope @azure/msal-common --scope @azure/msal-node",
-    "prepack": "npm run build:all",
+    "prepare": "npm install ../msal-common",
```

Comment:
Force installation of local `msal-common` package cause `lerna bootstrap` does not honor `file:*` deps.